### DISCUSSION TITLE
Noob with a question about language

### DISCUSSION THREAD

**Discussion by yengalvez** [mozilla/hubs#3723] [Hubs Cloud Questions] [answered]
Hi! I found a post with a json file to change the language of Hubs to spanish.

I have a AWS Cloud Hubs working but I don't know how to upload or use this file to change the language.

Thanks!

**Comment by robinkwilson** [comment, marked as answer]
Hello! The Hubs language on your Hubs Cloud should update automatically to the one matched to your settings in your browser.

As long as your visitors selected "Spanish" as the primary language, they will see it! Please let us know if that's not the case!

**Reply by yengalvez** [reply]
Mmm... I have spanish as primary language in my browser, and I created a AWS hubs cloud. I don't know if the automatic version of AWS isn't the last, but I can't see it in spanish and I don't know if there is an option to activate it :) thanks for your help. Do you know what can be the problem?=

**Reply by johnshaughnessy** [reply]
I don't think Spanish language support is available in Hubs Cloud (yet) but will be included in the next release (which will go out in about a week). (More details in https://github.com/mozilla/hubs/pull/3674 and https://github.com/mozilla/hubs/pull/3109 )

**Reply by yengalvez** [reply]
Amazing! How can I update the Hubs Cloud? Is there a tutorial or is it automatic?

**Reply by robinkwilson** [reply]
Automatic, without a custom client!

If you have a custom client, you'll need to merge in the latest changes from the hubs repo `hubs-cloud` branch into your custom client, then deploy them to your hub!

**Reply by yengalvez** [reply]
Thanks!!!!!
